components/camel-dhis2/LICENSE

```diff
@@ -0,0 +1,24 @@
+Copyright (c) 2004-2022, University of Oslo
```

Comment:
The code must be provided with an ASF License compliant license: https://www.apache.org/legal/resolved.html#category-a

As far as I see this project has been licensed as BSD 3-Clause

This copyright of University of Oslo shouldn't be there.